### ISSUE TITLE
WorkshopParent Quest needs resynced with UF4OP

### ISSUE THREAD

**Issue by Diskmaster** [kinggath/WorkshopFramework#121]
```Provisioners, Pack Brahmin, and Caravan Guards were retaining their names after reassignment due to a missing "Clears Name When Removed" flag on the aliases in the Workshop Parent quest. (Bug #19653)```

Trying to figure out why the WorkshopEnemyFaction was also altered.

**Comment by Diskmaster** [comment]
As per recent discussion in xEdit's discord, the newly added property is in reference to AFK 27621 and should also be present.

```WorkshopParentScript: The happiness malus applied to settlements as a penalty for the player killing a protected settler did also apply to synth infiltrators killed during a synth attack although they had their protected flags removed by the synth attack quest scripts (Bug #27621).```

**Comment by msalaba01** [comment]
```
[08/18/2022 - 06:37:51PM] warning: Property IgnoreAccuracyBonusInUI on script workshopparentscript attached to WorkshopParent (0002058E) cannot be initialized because the script no longer contains that property
```
WorkshopParent override in WSFW esm needs to be removed. This will allow UFO4P to correctly override WorkshopParent.

**Comment by Diskmaster** [comment]
That's what kinggath had mentioned in the discord, but I never got around to updating the bug.

**Comment by Diskmaster** [comment]
Fixed in 2.2.4
